src/index.ts

```diff
@@ -143,19 +155,22 @@ enum PropType {
 /**
  * Parse element content
  * @param content
+ * @param elementName
  */
-function parse (content: string): ParseResult | null {
+function parse (content: string, elementName: string): ParseResult | null {
   if (content.length === 0) return null
 
-  const parseRegExp = /^(?:\((.+?)\)){0,1}\s*\{(.+?)\}\s*(?:(.+))?/g
+  const parseRegExp = /^(?:\((.+?)\)){0,1}\s*\{(.+?)\}\s*(?:([^\s]*))?\s*(?:(.+))?/g
```

Comment:
Why this change?

Reply:
to capture the `description` and `field` properties properly (check the docs [here](https://apidocjs.com/#param-api-success) for a reference).

before we were capturing everything after the `{}` block and storing it in the `element` property and as far as I can tell we were actually even misusing it in the `setArrayElements` function.
I think now it's more clear

Reply:
now that I review it, I should add a default value to `field`.
right now I only do that for arrays (`setArrayElements` function)